libvast/vast/plugin.hpp

```diff
@@ -68,7 +77,8 @@ class plugin {
 public:
   /// The current version of the plugin API. When registering a plugin, set the
   /// corresponding plugin version in the `VAST_REGISTER_PLUGIN` macro.
-  constexpr static auto version = plugin_version{0, 1, 0, 0};
+  constexpr static auto version
+    = plugin_version{0, 1, 0, 0, VAST_VERSION, VAST_BUILD_TREE_HASH};
```

Comment:
Do we still need the `major` `minor` `patch` and `tweak` members in this struct? I guess it makes sense when we have some stability guarantees?

Reply:
For now I've opted to keep them, although they're pretty much useless until we make such guarantees. Shall I remove them for now?